src/OpenTelemetry.Contrib.Instrumentation.Runtime/RuntimeMetricsOptions.cs

```diff
@@ -0,0 +1,100 @@
+﻿// <copyright file="RuntimeMetricsOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Contrib.Instrumentation.Runtime
+{
+    /// <summary>
+    /// Options to define the runtime metrics.
+    /// </summary>
+    public class RuntimeMetricsOptions
+    {
+        /// <summary>
+        /// Gets or sets a prefiix used for the metric names.
+        /// </summary>
+        public string MetricPrefix { get; set; } = "process_runtime_dotnet_";
+
+        /// <summary>
+        /// Gets or sets a value indicating whether garbage collection metrics should be collected.
+        /// </summary>
+        public bool? GcEnabled { get; set; }
+
+#if NET6_0_OR_GREATER
+        /// <summary>
+        /// Gets or sets a value indicating whether jitter metrics should be collected.
+        /// </summary>
+        public bool? JitEnabled { get; set; }
+#endif
+
+#if NETCOREAPP3_1_OR_GREATER
+        /// <summary>
+        /// Gets or sets a value indicating whether threading metrics should be collected.
+        /// </summary>
+        public bool? ThreadingEnabled { get; set; }
+#endif
+
+        /// <summary>
+        /// Gets or sets a value indicating whether performance metrics should be collected.
+        /// </summary>
+        public bool? PerformanceEnabled { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether assembly metrics should be collected.
+        /// </summary>
+        public bool? AssembliesEnabled { get; set; }
+
+        /// <summary>
+        /// Gets a value indicating whether all metrics are enabled.
+        /// </summary>
+        internal bool IsAllEnabled => this.GcEnabled == null
```

Comment:
have you considered making everything opt-in? i.e by default, user gets nothing by simply adding AddRunTimeMetrics(). One needs to explicitly enable individually to get them.

Reply:
So then the `configure` parameter in `MeterProviderBuilderExtensions.AddRuntimeMetrics` should be mandatory, right?. Otherwise it makes no sense to add AddRuntimeMetrics().

Reply:
we could make some of them as enabled by default.

Reply:
@cijothomas ok, which one should be enabled by default? "Performance"?

Reply:
pick anything random for now or make all Enabled by default. We can merge and get a nuget out, and start collecting some feedback, and adjust accordingly.